tests/rafter/pkg/mockice/mockice.go

```diff
@@ -0,0 +1,219 @@
+package mockice
```

Comment:
Copied from cbs almost entirely, just for a few changes, like https://github.com/kyma-project/kyma/pull/6569/files#diff-76b6edad07d9c5a172c2e149112fe70dR57-R63

Reply:
and https://github.com/kyma-project/kyma/pull/6569/files#diff-76b6edad07d9c5a172c2e149112fe70dR184-R199